libraries/AP_InertialSensor/AP_InertialSensor_Invensense.h

```diff
@@ -125,19 +125,19 @@ class AP_InertialSensor_Invensense : public AP_InertialSensor_Backend
     // use a sample counter to "timestamp" samples from the sensor
     // Since the system timebase is not synchronized to the sensor sample clock,
     // this is the best way to identify missing samples in the log. With this
-    // counter supplied to the notify methods, the SampleUS field becomes an integer
+    // counter supplied to the notify methods, the SampleC field becomes an integer
     // sequence which increments each time a sample is read from the FIFO. As long as
     // the FIFO is read at a sufficient rate, there should be no dropped samples
     // between the sensor and the call to notify, with the exception of FIFO corruption:
     // if corruption is detected, _seqcnt is zeroed to indicate a break in continuity.
     uint32_t _seqcnt;
 
-#define INVENSENSE_DEBUG 1
-//#undef INVENSENSE_DEBUG
+//#define INVENSENSE_DEBUG 1
+#undef INVENSENSE_DEBUG
```

Comment:
These two lines just shouldn't be here, or in the "worst" case, leave just the define commented. Forcing an undef here means you can't define this macro in the command-line.